Source/Scene/Model/Model.js

```diff
@@ -1102,6 +1102,7 @@ Object.defineProperties(Model.prototype, {
       }
       //>>includeEnd('debug');
 
+      updateBoundingSphereAndScale(this);
```

Comment:
In `updateBoundingSphereAndScale` there's the line:

```
  if (!model._updateModelMatrix && model._minimumPixelSize === 0.0) {
    return;
  }
```

If `_updateModelMatrix` is false the bounding sphere won't be updated. Here's a sandcastle that shows when that could happen: [sandcastle](http://localhost:8080/Apps/Sandcastle/index.html#c=zVNtb9MwEP4rVj6lUucMDb6MrIIVCT4wMdGKT5HQxbm0Fo5d2ZeOgvbfOdsZtFt/AFVl+d6e5+7xRTkbSOw1PqAXN8Lig1hi0OMgvyVf2RQq2UtnCbRF3xRz8buxQmyMa/Fa9GACNvZx9raxKqEFhRYZLKPKZHLwKTy4Dg2Hk1/uvB406T0GCV1XTuR3MUf23g0fDfVl4hu9uRZNIWXF/xUMO4MfgKBKeKHKle+1n27f+So3puV+Y3ez1EHKlR6hO9wzuA4oaYu27EerSDtbpoRZHjC36/o+IJ1q84kBtN3ca1Lbr2A3WF5cycu5uIzHGz4jm5gmVDCgBxnVuHWjjYWr3RY9ZjLZnjjnE2FEeMxNr8B2CgIZjBKtnTMt+NuRiPttirUHGwwQCk4THpmT+C0za3ysv7OdjEVTGQd4tnOdSM9DjiFNkuPpvAPy+ifXPD1Vsl+nx1r/Ay2P5FqCJ76BvSovjngnvaJcWa//Q/BiXtSBDgYXETL+3ulh5zzFDSx59wh591jwULWj+oEkVQiZX4i6Oi6tO70Xurs58xEJZSAEjvSjMSv9C5tiUVec/6LUuDT7lz16A4eYtn21+JydUsq6YvN8JeVVeYb8Bw)